src/table/utils/styling-utils.ts

```diff
@@ -1,16 +1,30 @@
 import { stardust } from "@nebula.js/stardust";
 import { PAGINATION_HEIGHT } from "@qlik/nebula-table-utils/lib/constants";
 import type { ExtendedTheme } from "@qlik/nebula-table-utils/lib/hooks/use-extended-theme/types";
-import { COLORING, isDarkColor, removeOpacity, toRGB } from "@qlik/nebula-table-utils/lib/utils";
+import { COLORING, getHoverColor, isDarkColor, removeOpacity, toRGB } from "@qlik/nebula-table-utils/lib/utils";
 import { ContentStyling, HeaderStyling, PaletteColor, TableLayout } from "../../types";
 import { SelectionStates } from "../constants";
 import { SELECTION_STYLING } from "../styling-defaults";
-import { CellStyle, GeneratedStyling } from "../types";
+import { CellStyle, FeatureFlags, GeneratedStyling } from "../types";
 
 export const LINE_HEIGHT = 4 / 3;
 export const CELL_PADDING_HEIGHT = 8;
 export const CELL_BORDER_HEIGHT = 1;
 
+// TODO: maybe shared repo?
+const HEADER_MENU_COLOR_MODIFIER = {
```

Comment:
If they use the same modifiers maybe that should be the default?

Reply:
Yes, it can be default, but we also need to have a way to override them, just in case if we want it to